test/models/responses/CashAccountStatementResponseContainerSpec.scala

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.responses
+
+import play.api.libs.json.Json
+import utils.SpecBase
+
+
+class CashAccountStatementResponseSpec extends SpecBase {
+
+  "CashAccountStatementResponseContainer" should {
```

Comment:
Better to say in terms of Json Reads and Writes like below as straight away tells what the test is for Reads or Writes. Same for CashAccountStatementErrorResponse

"CashAccountStatementResponseContainer Reads" or "CashAccountStatementResponseContainer Writes"